test/index.html

```diff
@@ -11,6 +11,8 @@
 			WCT.loadSuites([
 				'../components/button/test/button-icon.html',
 				'../components/button/test/button-subtle.html',
+				'../components/button/test/button.html',
```

Comment:
Noticed I had not added this with the buttons PR, I've added it here made slight change to buttons test to make it all work.